fe/fe-core/src/main/java/org/apache/doris/planner/OlapScanNode.java

```diff
@@ -1044,6 +1045,7 @@ private void filterDeletedRows(Analyzer analyzer) throws AnalysisException {
             Expr conjunct = new BinaryPredicate(BinaryPredicate.Operator.EQ, deleteSignSlot, new IntLiteral(0));
             conjunct.analyze(analyzer);
             conjuncts.add(conjunct);
+            closePreAggregation(Column.DELETE_SIGN + " is used as conjuncts.");
```

Comment:
MoW requires isPreAggregation on, should use `setIsPreAggregation(false, reason)`?